scripts/uosc.lua

```diff
@@ -4172,15 +4172,18 @@ end))
 mp.observe_property('demuxer-cache-state', 'native', function(prop, cache_state)
 	local cached_ranges = cache_state and cache_state['seekable-ranges'] or {}
 	local uncached_ranges = nil
-	if state.duration and state.is_stream then
+	if state.duration then
```

Comment:
Wouldn't it be simpler to just check for `#cached_ranges > 0` here instead of counting in the for?

Reply:
I'm not a Lua programmer, I found this method when I searched the web how to get the table size, I'll change it when I wake up.